[2024-11-07T09:05:53.027+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: openweatherdata_dag.group_4.task_4b manual__2024-11-07T09:05:38.981136+00:00 [queued]>
[2024-11-07T09:05:53.047+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: openweatherdata_dag.group_4.task_4b manual__2024-11-07T09:05:38.981136+00:00 [queued]>
[2024-11-07T09:05:53.049+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-11-07T09:05:53.125+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): group_4.task_4b> on 2024-11-07 09:05:38.981136+00:00
[2024-11-07T09:05:53.150+0000] {standard_task_runner.py:60} INFO - Started process 668 to run task
[2024-11-07T09:05:53.168+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'openweatherdata_dag', 'group_4.task_4b', 'manual__2024-11-07T09:05:38.981136+00:00', '--job-id', '165', '--raw', '--subdir', 'DAGS_FOLDER/model_meteo.py', '--cfg-path', '/tmp/tmpg0w_43hd']
[2024-11-07T09:05:53.181+0000] {standard_task_runner.py:88} INFO - Job 165: Subtask group_4.task_4b
[2024-11-07T09:05:53.260+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2024-11-07T09:05:53.448+0000] {task_command.py:423} INFO - Running <TaskInstance: openweatherdata_dag.group_4.task_4b manual__2024-11-07T09:05:38.981136+00:00 [running]> on host 8a2ba9ce15fe
[2024-11-07T09:05:53.758+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='openweatherdata_dag' AIRFLOW_CTX_TASK_ID='group_4.task_4b' AIRFLOW_CTX_EXECUTION_DATE='2024-11-07T09:05:38.981136+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-11-07T09:05:38.981136+00:00'
[2024-11-07T09:05:53.993+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-11-07T09:05:54.049+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=openweatherdata_dag, task_id=group_4.task_4b, execution_date=20241107T090538, start_date=20241107T090553, end_date=20241107T090554
[2024-11-07T09:05:54.176+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-11-07T09:05:54.269+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
